src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -981,33 +988,34 @@ It is also possible to change the ProActive Nodes Account using the 'Change Acco
 
 When you click on 'GUI Edit', the following window appears:
 
-image::config_editor_general.png[align="center"]
+image::config_editor_general.png[align=center, width=612, height=306]
 
 In the general tab, the user can specify:
 
 * The *ProActive Scheduler* location.
-* The *JRE* location (usually something like `C:\Program Files\Java\jdk1.6.0_12`).
+* The *JRE* location (usually something like `C:\Program Files\ProActive Agent\schedworker\jre`).
```

Comment:
`usually something like ` =>  Not clear nor professional. Is ProActive coming w/ its JRE ?

Reply:
Yes it is.

Reply:
windows agent is a x86 application, its default installation path is C:\Program Files (x86)\ProActiveAgent